### ISSUE TITLE
Add pagination or lazy-load to results table

### ISSUE THREAD

**Issue by ITAYC0HEN** [CERT-Polska/mquery#39]
**Description**
When executing a query with many results, the interface will show all the matches in the results table. The table can be with tens of thousands of results for a big dataset, which will make the page slow to response. I suggest implementing pagination (the xhr requests are already fetching 50 at a time) or a [lazy-loading](https://en.wikipedia.org/wiki/Lazy_loading)

**Comment by msm-code** [comment]
Yeah, that can be quite a pain

**Comment by msm-code** [comment]
To test:

1. Fenerate a few test files:

```python
import string
import os
import random

for i in range(40000):
    print(i)
    data = os.urandom(40000)
    open(os.urandom(16).hex(), 'wb').write(data)
```

2. index them all

3. Do some trivial query like:

```
rule hmm {
strings: $a = "a"
condition: $a
}
```

4. Observe that the query finishes quickly, but loading all the results takes ages.

**Comment by ITAYC0HEN** [comment]
#78 is somewhat related to the planning of the new table. A modern approach to a filterable table.

**Comment by msm-code** [comment]
fixed with #96
